quinn-proto/src/connection.rs

```diff
@@ -3245,6 +3253,9 @@ pub enum ConnectionEvent {
 /// Events to be sent to the Endpoint
 #[derive(Clone, Debug)]
 pub enum EndpointEvent {
+    Closed {
+        remote: SocketAddr,
```

Comment:
Could this be moved into `ConnectionMeta`?

Reply:
I don't think that makes sense, `Connection` actually has a number of mechanics around the `remote`.

Reply:
It makes sense for `Connection` to *know about* the `remote`, but it seems a little odd for `Endpoint` not to retain a copy, when it's using it as a key in its internal tables. Not a big deal, though.